ui/app/AppLayouts/Communities/panels/MembersTabPanel.qml

```diff
@@ -99,34 +94,37 @@ Item {
                             text: model.membershipRequestState === Constants.CommunityMembershipRequestState.KickedPending ? qsTr("Kick pending") : qsTr("Kick")
                             type: StatusBaseButton.Type.Danger
                             size: StatusBaseButton.Size.Small
-                            onClicked: root.kickUserClicked(model.pubKey, model.displayName)
+                            onClicked: root.kickUserClicked(model.pubKey, memberItem.title)
                             enabled: kickButton.interactive
                         }
                     },
 
                     DisabledTooltipButton {
                         id: banButton
+                        anchors.verticalCenter: parent.verticalCenter
                         //using opacity instead of visible to avoid the acceptButton jumping around
                         opacity: banVisible
                         interactive: banEnabled
```

Comment:
I'm fixing the same issue here, but using the enabled property instead.
https://github.com/status-im/status-desktop/pull/11920

Reply:
Alright, I still believe the other commit for `StatusTooltip` makes sense though

Reply:
@alexjba your turn now, feel free to fix/resolve it as you like ;)